src/System.Memory/src/System.Memory.csproj

```diff
@@ -22,7 +22,9 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Release|AnyCPU'" />
   <ItemGroup>
+    <Compile Include="System\Buffers\ReadOnlySequence_TryGet.cs" />
```

Comment:
nit: ordering

Can you please move these to the related ItemGroups.

Reply:
This is why globbing is better